.bazelci/presubmit.yml

```diff
@@ -4,22 +4,48 @@ tasks:
   ubuntu1604:
     name: ":linux:"
     shell_commands:
-    - yes | ./configure
+    - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+    - bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
+    - source $HOME/miniconda/bin/activate
+    - wget https://raw.githubusercontent.com/plaidml/plaidml/master/environment.yml
+    - $HOME/miniconda/bin/conda env update -f ./environment.yml
+    - conda activate $HOME/miniconda
     build_targets:
     - "//plaidml/keras:wheel"
+    build_options:
+    - "--config=linux_x86_64"
     test_targets:
     - "..."
+    test_options:
+    - "--cpu=linux_x86_64"
   macos:
     name: ":apple:"
     shell_commands:
-    - yes | ./configure
+    - curl -O https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh 
+    - sh ./Miniconda3-latest-MacOSX-x86_64.sh -b -p $HOME/miniconda
+    - curl -O https://raw.githubusercontent.com/plaidml/plaidml/master/environment.yml
+    - $HOME/miniconda/bin/conda env update -f ./environment.yml
+    - conda activate $HOME/miniconda
     build_targets:
     - "//plaidml/keras:wheel"
+    build_options:
+    - "--config=windows_x86_64"
```

Comment:
You probably meant `- --config=macos_x86_64` here

Reply:
Yup, I've fixed this in commit [`814cd24`](https://github.com/plaidml/plaidml/pull/435/commits/814cd246c8e9fee003c7abba4b074f7e7dc9259d)